iiod/init/iiod.service.cmakein

```diff
@@ -2,7 +2,7 @@
 #
 # iiod - Systemd init script
 #
-# Copyright (C) 2016-2021 Analog Devices Inc.
+# Copyright (C) 2016-2024 Analog Devices Inc.
```

Comment:
Kind on unrelated change. You could have mentioned the change in the commit message....